README.md

```diff
@@ -2,7 +2,7 @@
 [![Build Status](https://travis-ci.com/camel-tooling/vscode-camelk.svg?branch=master)](https://travis-ci.com/camel-tooling/vscode-camelk)
 [![CircleCI](https://circleci.com/gh/camel-tooling/vscode-camelk.svg?style=shield)](https://circleci.com/gh/camel-tooling/vscode-camelk)
 [![License](https://img.shields.io/badge/license-Apache%202-blue.svg)]()
-[![Gitter](https://img.shields.io/gitter/room/camel-tooling/Lobby.js.svg)](https://gitter.im/camel-tooling/Lobby)
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://camel.zulipchat.com/#narrow/stream/258729-camel-tooling)
```

Comment:
Would just "Chat" be a better name?

Reply:
I agree with Lars here

Reply:
or even just "Zulip" or "Zulip Chat" to match the old "Gitter" link

Reply:
updated to "Zulip chat"